lib/vscode/.eslintignore

```diff
@@ -16,3 +16,8 @@
 **/extensions/markdown-language-features/notebook-out/**
 **/extensions/typescript-basics/test/colorize-fixtures/**
 **/extensions/**/dist/**
+# These are code-server code symlinks.
+src/vs/base/node/proxy_agent.ts
+src/vs/ipc.d.ts
```

Comment:
`src/vs/server/common/ipc.ts` now I think.

Reply:
Actually maybe we don't need this line? The linter didn't seem to complain. Maybe because it's a `.d.ts`? I wonder if we should move it into `typings`.

Reply:
Not sure if we should move into `typings`, but I moved it to `src/vs/base/common` (and updated the path.)